Dalamud/Dalamud.cs

```diff
@@ -116,14 +117,16 @@ public sealed class Dalamud : IDisposable {
                 this.Data = new DataManager(this.StartInfo.Language);
                 await this.Data.Initialize(this.baseDirectory);
 
+				// TODO: better way to do this?  basically for lumina injection
+            	SeString.Dalamud = this;
```

Comment:
I think just giving DataMgr to each payload is a bit nicer, will do that after merge